src/FourBitTwoDisclosureDeviceUnlocker.java

```diff
@@ -38,7 +47,7 @@
     private static CharSequence peekedPattern = null;
 
     /** Char we are changing device bits to default to 'T'*/
-    private static char changeBitTo = 'T';
+    private static char changeBitTo = TRUE;
```

Comment:
This should help us scale the project in the future if he decides to make his `Device` use different characters instead of `'T'` and `'F'`.